docs/guide/installation.rst

```diff
@@ -88,7 +88,7 @@ It is possible to select which "extra" dependencies you want to install, if you
 
 The available options are: ``[asdf]``, ``[dask]``,``[database]``,``[instr]``,``[image]``,``[jpeg2000]``,``[map]``,``[net]``,``[timeseries]``,``[visualization]``.
```

Comment:
Just noticed above the broken link I fixed that these options are not being formatted correctly (https://docs.sunpy.org/en/latest/guide/installation.html#pip). Adding a space after each comma fixes it and docs build successfully on my computer. Should I add that fix to this PR too or create a new one?

Reply:
Add it to this PR if you could.

Reply:
Done

Reply:
Thanks.